fluent-react/test/localized_overlay_test.js

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import { MessageContext } from '../../fluent/src';
+import ReactLocalization from '../src/localization';
+import { Localized } from '../src/index';
+
+suite('Localized - overlay', function() {;
+
+  test('one element is matched', function() {
+    const mcx = new MessageContext();
+    const l10n = new ReactLocalization([mcx]);
+
+    mcx.addMessages(`
+foo = Click <button>me</button>!
+`)
+
+    const wrapper = shallow(
+      <Localized id="foo" button={<button onClick={alert}></button>}>
+        <div />
+      </Localized>,
+      { context: { l10n } }
+    );
+
+    assert.ok(wrapper.contains(
+      <div>
+        Click <button onClick={alert}>me</button>!
+      </div>
+    ));
+  });
+
+  test('two elements are matched', function() {
+    const mcx = new MessageContext();
+    const l10n = new ReactLocalization([mcx]);
+
+    mcx.addMessages(`
+foo = <confirm>Sign in</confirm> or <cancel>cancel</cancel>.
+`)
+
+    const wrapper = shallow(
+      <Localized id="foo"
+        confirm={<button className="confirm"></button>}
+        cancel={<button className="cancel"></button>}
```

Comment:
I like this because it's very semantic, but I'm slightly afraid that localizers also translate the names of elements, like so:

    <confirm>Sign in</confirm> or <cancel>cancel</cancel>.
    <potwierdź>Zaloguj się</potwierdź> albo <anuluj>anuluj</anuluj>.

For this reason I think it's safest to use known HTML names, like `button` or `a`.